docs/index.adoc

```diff
@@ -49,7 +49,7 @@ Picocli is a one-file framework for creating Java command line applications with
 It supports a variety of command line syntax styles including POSIX, GNU, MS-DOS and more. It generates highly customizable usage help messages that use <<ANSI Colors and Styles,ANSI colors and styles>> to contrast important elements and reduce the cognitive load on the user.
 
 Picocli-based applications can have link:autocomplete.html[command line TAB completion] showing available options, option parameters and subcommands, for any level of nested subcommands.
-Picocli-based applications can be ahead-of-time compiled to a image:https://www.graalvm.org/resources/img/logo-colored.svg[GraalVM]
+Picocli-based applications can be ahead-of-time compiled to an image:https://www.graalvm.org/resources/img/logo-colored.svg[GraalVM]
```

Comment:
I think the original is actually okay: it should get rendered as "a native image".
(The `image:https:...` is to embed an image in the text. The word "image" is not part of the text.)

Reply:
You are right, I overlooked the fact that this is a rendered image.
Reverted as requested.